src/test/groovy/ee/tuleva/onboarding/account/AccountStatementServiceSpec.groovy

```diff
@@ -57,8 +56,12 @@ class AccountStatementServiceSpec extends Specification {
     def activeNonZeroFund = FundBalanceDto.builder().isin("4").value(ONE).activeContributions(true).build()
 
     episService.getAccountStatement(person) >> [nonActiveZeroFund, nonActiveNonZeroFund, activeZeroFund, activeNonZeroFund]
-    fundBalanceConverter.convert(_, person) >> { FundBalanceDto fundBalanceDto, _ ->
-      FundBalance.builder().fund(Fund.builder().isin(fundBalanceDto.isin).build()).build()
+    fundBalanceConverter.convert(_, _) >> { FundBalanceDto fundBalanceDto, _ ->
+      FundBalance.builder()
+        .fund(Fund.builder().isin(fundBalanceDto.isin).build())
```

Comment:
Not the cleanest way to mock out the converter behaviour here, but will improve in the future.